.tekton/tsa-unit-test.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: go-unit-test
+  annotations:
+    tekton.dev/title: "Go Unit Test Task"
+spec:
+  workspaces:
+    - name: source
+  steps:
+    - name: run-tests
+      image: registry.access.redhat.com/ubi9/go-toolset@sha256:c3a9c5c7fb226f6efcec2424dd30c38f652156040b490c9eca5ac5b61d8dc3ca
+      workingDir: $(workspaces.source.path)/source
+      script: |
+        #!/usr/bin/env sh
+        CGO_ENABLED=0 go build -buildvcs=false -trimpath -o bin/timestamp-cli ./cmd/timestamp-cli
+        CGO_ENABLED=0 go build -buildvcs=false -trimpath -o bin/timestamp-server ./cmd/timestamp-server
+        go test ./...
+        
+# This file bundles the unit tests for the timestamp-authority. 
+# If any changes are made to this file, it must be pushed to Quay using the following command:
+# 'tkn bundle push quay.io/securesign/tsa-unit-test:latest -f .tekton/tsa-unit-test.yaml'.
+# This will generate a new SHA for the bundle. 
+# Ensure that this new SHA is updated in the pull and push pipeline files for each component.
```

Comment:
Minor suggestion to remove this text entirely since we're using `latest` for the test

Reply:
Done, I've also updated my other prs :)